src/test/lib/QuicDrill.cpp

```diff
@@ -215,11 +212,14 @@ struct DrillSender {
         //
         memcpy(SendBuffer->Buffer, PacketBuffer->data(), DatagramLength);
 
+        CXPLAT_ROUTE Route;
+        CxPlatSocketGetLocalAddress(Binding, &Route.LocalAddress);
+        Route.RemoteAddress = ServerAddress;
```

Comment:
Why move this down? I *believe* Yi's change had to move some of these **up** so this might cause conflicts. I recommend just putting these up.

Reply:
Ditto with all the other ones below.

Reply:
The idea was to follow the convention of declaring variables as near as possible to where they're used. Will move back up here and elsewhere to reduce merge conflicts.

Reply:
My change is already in. There should be no merge conflicts.

Reply:
Oh, this is going to main. Nvm.

Reply:
Actually, should this entire change go into nibanks/dpdk first and then we can bring both API changes up to main? Or are you going to first get this in main and resolve conflicts when merging to dpdk?

Reply:
Yeah, the plan is to merge main into nibanks/dpdk after checking this in.

Reply:
Seems like I made a mistake not checking into main for the API changes first. Now, bringing them up will be difiicult.